core/src/main/scala/scalaz/std/Future.scala

```diff
@@ -5,7 +5,8 @@ import scala.concurrent.{Await, CanAwait, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
 trait FutureInstances1 {
-  implicit def futureInstance(implicit ec: ExecutionContext): Monad[Future] with Cobind[Future] =
+
+  implicit def futureInstance(implicit ec: ExecutionContext): Cobind[Future] with MonadError[({type λ[α,β] = Future[β]})#λ, Throwable] with Catchable[Future] =
```

Comment:
@vmarquez This error should be except-able, because we only support the three `import` lines @xuwei-k mentions, and all these objects will be recompiled to include the method you've added.